Userland/Libraries/LibThreading/Thread.cpp

```diff
@@ -15,7 +15,7 @@ Threading::Thread::Thread(Function<intptr_t()> action, StringView thread_name)
     , m_thread_name(thread_name.is_null() ? ""sv : thread_name)
 {
     register_property("thread_name", [&] { return JsonValue { m_thread_name }; });
-    register_property("tid", [&] { return JsonValue { m_tid }; });
+    register_property("tid", [&] { return JsonValue { bit_cast<FlatPtr>(m_tid) }; });
```

Comment:
For nicer output, how about calling [`pthread_threadid_np`](https://www.manpagez.com/man/3/pthread_threadid_np/) on macOS?

Reply:
Hm. I went with the #ifdef solution to avoid thinking about all the fun BSD-isms for the quick "get CI working again" fix. If you want to do  a follow-up PR for that and to properly do pthread_set_name_np on BSD and macOS here and the other places we do pthread_setname_np it might be nice :)

Reply:
.... oooor I could do that now b/c my solution doesn't work in Ubuntu linux even though I tested it on my linux box (I thought)

Reply:
Meh. I'll just ignore BSD-like platforms for now and keep a fixme.